File: src/test/java/com/github/krystianmuchla/skyr/DatabaseContainer.java
package com.github.krystianmuchla.skyr;

import org.testcontainers.containers.MySQLContainer;

public class DatabaseContainer extends MySQLContainer<DatabaseContainer> {
    public DatabaseContainer() {
        super("mysql:8.1.0");
    }

    @Override
    public void start() {
        super.start();
        System.setProperty("SKYR_DATABASE_URL", getJdbcUrl());
        System.setProperty("SKYR_DATABASE_USER", getUsername());
        System.setProperty("SKYR_DATABASE_PASSWORD", getPassword());
    }
}

File: src/test/kotlin/com/example/skyr/DatabaseContainer.kt
package com.example.skyr

import org.testcontainers.containers.MySQLContainer

class DatabaseContainer : MySQLContainer<DatabaseContainer>("mysql:8.1.0") {

    override fun start() {
        super.start()
        System.setProperty("SKYR_DATABASE_URL", jdbcUrl)
        System.setProperty("SKYR_DATABASE_USER", username)
        System.setProperty("SKYR_DATABASE_PASSWORD", password)
    }
}
